tests/e2e/performance/csi_tests/test_pvc_snapshot_performance.py

```diff
@@ -250,7 +250,7 @@ def test_pvc_snapshot_performance(self, pvc_size):
             self.pod_object.fillup_fs(size=file_size, fio_filename=file_name)
 
             # Wait for fio to finish
-            fio_result = self.pod_object.get_fio_results()
+            fio_result = self.pod_object.get_fio_results(timeout=3600)
```

Comment:
This is default fio profile right? hope we validated properly for the timeout. 
I have faced something similar in OOM kill TC fio didn't complete in expected time. Eventually I stopped using this func and instead used perf image and the custom profile so there will be more control on handling the IO, again this is just a suggestion will leave it to you on how you want it.

Reply:
@ramkiperiy  - yes, we run default FIO load, and with the increased timeout it works ok. 
This is now it works in all the previous versions as well, I wouldn't change the IO type since it might affect the performance comparison.